attributes/default.rb

```diff
@@ -42,8 +42,15 @@
 default['ntp']['statsdir'] = '/var/log/ntpstats/'
 default['ntp']['conf_owner'] = 'root'
 default['ntp']['conf_group'] = 'root'
-default['ntp']['var_owner'] = 'ntp'
-default['ntp']['var_group'] = 'ntp'
+
+if platform?('debian') && node['platform_version'].to_i <= 12
```

Comment:
```suggestion
if platform?('debian') && node['platform_version'].to_i < 12
```

Shouldn't this be less than? Might need more coffee